### PR TITLE
docs: add dockerub descriptions

### PR DESCRIPTION
**Pull request by bill-mahoney**
**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [ ] I am not introducing a new dependency (add notes below if you are)
- [ ] I have added unit tests for the new feature or bug fix (if not, why?)
- [ ] I have fully tested (add details below) this the new feature or bug fix (if not, why?)
- [ ] I have opened a PR for the related docs change (if not, why?)
  <link to docs PR>

### CODE REVIEW

File: image-overview-templates/new-names/core-common-config-bootstrapper-arm64.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+$$./common/version-minnesota.md |
+        - [Dockerfile](https://github.com/edgexfoundry/edgex-go/blob/v3.0.0/cmd/core-common-config-bootstrapper/Dockerfile)
+
+$$./common/quick-reference-cont-edgex-go.md
+
+$$./common/what-is-edgex.md
+
+# What's in this image?
+
```
Comment (lenny-intel) [inline image-overview-templates/new-names/core-common-config-bootstrapper-arm64.md:15]: This image contains the **arm64** version of the service that seeds the Configuration Provider with the configuration common to all services

File: image-overview-templates/new-names/core-common-config-bootstrapper.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+$$./common/version-minnesota.md |
+        - [Dockerfile](https://github.com/edgexfoundry/edgex-go/blob/v3.0.0/cmd/core-common-config-bootstrapper/Dockerfile)
+
+$$./common/quick-reference-cont-edgex-go.md
+
+$$./common/what-is-edgex.md
+
+# What's in this image?
+
+
```
Comment (lenny-intel) [inline image-overview-templates/new-names/core-common-config-bootstrapper.md:16]: This image contains the service that seeds the Configuration Provider with the configuration common to all services

File: image-overview-templates/new-names/security-proxy-auth-arm64.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+$$./common/version-minnesota.md |
+        - [Dockerfile](https://github.com/edgexfoundry/edgex-go/blob/v3.0.0/cmd/security-proxy-auth/Dockerfile)
+
+$$./common/quick-reference-cont-edgex-go.md
+
+$$./common/what-is-edgex.md
+
+# What's in this image?
+
```
Comment (lenny-intel) [inline image-overview-templates/new-names/security-proxy-auth-arm64.md:15]: add `**arn64** version of the ` to @bnevis-i  description for the non-arm version

File: image-overview-templates/new-names/security-proxy-auth.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+$$./common/version-minnesota.md |
+        - [Dockerfile](https://github.com/edgexfoundry/edgex-go/blob/v3.0.0/cmd/security-proxy-auth/Dockerfile)
+
+$$./common/quick-reference-cont-edgex-go.md
+
+$$./common/what-is-edgex.md
+
+# What's in this image?
+
```
Comment (lenny-intel) [inline image-overview-templates/new-names/security-proxy-auth.md:15]: @bnevis-i , please provide short description of this service image

i.e. this image contains the service that ....

**Review by lenny-intel — CHANGES_REQUESTED** [review]

File: image-overview-templates/new-names/security-proxy-auth.md (hunk 4 above)
Comment (bnevis-i) [reply image-overview-templates/new-names/security-proxy-auth.md:15]: "This image was added in EdgeX 3.0 to perform delegated authentication for the NGINX API gateway.  It evaluates the incoming JWT on API gateway calls and returns success (HTTP 200) or failure (HTTP 401) to authorize or deny the request from being forwarded to the backend EdgeX microservice."

File: image-overview-templates/new-names/security-proxy-auth-arm64.md (hunk 3 above)
Comment (bnevis-i) [reply image-overview-templates/new-names/security-proxy-auth-arm64.md:15]: **arm64**

File: image-overview-templates/new-names/security-proxy-auth.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
```
Comment (bnevis-i) [inline image-overview-templates/new-names/security-proxy-auth.md:16]: Forgot to add the description.

**Review by bnevis-i — APPROVED** [review]

File: image-overview-templates/new-names/security-proxy-auth-arm64.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+This **arm64** image was added in EdgeX 3.0 to perform delegated authentication for the NGINX API gateway. It evaluates the incoming JWT on API gateway calls and returns success (HTTP 200) or failure (HTTP 401) to authorize or deny the request from being forwarded to the backend EdgeX microservice.
```
Comment (lenny-intel) [inline image-overview-templates/new-names/security-proxy-auth-arm64.md:16]: @bnevis-i , these descriptions are not version specific.

```suggestion
This image contains the **arm64** version of the service which performs delegated authentication for the NGINX API gateway. It evaluates the incoming JWT on API gateway calls and returns success (HTTP 200) or failure (HTTP 401) to authorize or deny the request from being forwarded to the backend EdgeX microservice.
```

File: image-overview-templates/new-names/security-proxy-auth.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+$$./common/version-minnesota.md |
+        - [Dockerfile](https://github.com/edgexfoundry/edgex-go/blob/v3.0.0/cmd/security-proxy-auth/Dockerfile)
+
+$$./common/quick-reference-cont-edgex-go.md
+
+$$./common/what-is-edgex.md
+
+# What's in this image?
+
+This image was added in EdgeX 3.0 to perform delegated authentication for the NGINX API gateway. It evaluates the incoming JWT on API gateway calls and returns success (HTTP 200) or failure (HTTP 401) to authorize or deny the request from being forwarded to the backend EdgeX microservice.
```
Comment (lenny-intel) [inline image-overview-templates/new-names/security-proxy-auth.md:16]: @bnevis-i , these descriptions are not version specific.

```suggestion
This image contains the service which performs delegated authentication for the NGINX API gateway. It evaluates the incoming JWT on API gateway calls and returns success (HTTP 200) or failure (HTTP 401) to authorize or deny the request from being forwarded to the backend EdgeX microservice.
```

**Review by lenny-intel — CHANGES_REQUESTED** [review]

**Review by lenny-intel — APPROVED** [review]

**Pull request merged**